src/server/network/protocol/protocolgame.cpp

```diff
@@ -719,6 +719,10 @@ void ProtocolGame::parsePacket(NetworkMessage &msg) {
 
 void ProtocolGame::parsePacketDead(uint8_t recvbyte) {
 	if (recvbyte == 0x14) {
+		// Remove player from game if click "ok" using otcv8
+		if (player && otclientV8 > 0) {
+			g_game().removePlayerUniqueLogin(player->getName());
```

Comment:
This is something I really don't know, but I think it should cause yes. I added this because @hriqueXimenes reported (https://github.com/opentibiabr/canary/pull/1109#pullrequestreview-1485285751) me the wheel pull request problem, after investigating I found that otcv8 "removes" the creature after it dies by pressing "ok" so we should also do the same here.

Reply:
Client CIP does not remove the player when he dies, you are still logged in until you choose an option on the death screen...

Removing in the cip client also allows the duplication of characters and more than 1 client

Reply:
ahh, i see, that makes sense